data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
```

Comment:
>I would be less inclined to play it if it were more expensive to play than my fw salary and tribute from Pirate planets.

I don't want to completely knock the Bactrian down to never be used again, or to rarely be used due to the crew costs, I only want to knock it down enough as to make other warships more reasonable to use. And although this wasn't totally intended when balancing this, the Bactrian is really a middle ground between civilian heavy warships and military capital warships, and that 70 crew requirement is right now in that spot.
Leviathan: 45
Falcon: 52
Bactrian: 70
Kestrel: 72
Cruiser: 81